src/radical/pilot/utils/prof_utils.py

```diff
@@ -496,8 +497,9 @@ def get_session_profile(sid, src=None):
 
     if os.path.exists(src):
         # we have profiles locally
-        profiles  = glob.glob("%s/*.prof"   % src)
-        profiles += glob.glob("%s/*/*.prof" % src)
+        profiles  = glob.glob("%s/*.prof"     % src)
+        profiles += glob.glob("%s/*/*.prof"   % src)
+        profiles += glob.glob("%s/*/*/*.prof" % src)
```

Comment:
would it be ok not to limit the number of levels to check?
`profiles  = glob.glob("%s/**/*.prof"     % src,  recursive=True)`

Reply:
I didn't know `glob` supports that - thanks! :-)
Having said that, I would like to be carful with that.  if `rp_install` exists in the pilot sandbox, then the above can become costly.  
I am not too happy with the above either though - this now fetches more than it should (which takes time) - but otherwise we miss out on the raptor profiles which live in the master sandboxes. With the termination nowadays more relying on a hard kill, the bootstrapper usually has no chance to tar up the sandboxes and the client can't pull that tarball anymore.  We probably should initiate that tar from the client side as a separate saga job - but that would be out of scope for this PR.
I'll open an issue for this - #2663

Reply:
I applied the `**` syntax now - but also, the fetching was fixed in `rpu.session` lately.